biscuit-parser/src/parser.rs

```diff
@@ -1318,14 +1315,14 @@ mod tests {
 
         assert_eq!(
             super::expr("1 > 2 == 3").map(|(i, o)| (i, o.opcodes())),
-            Err(nom::Err::Failure(crate::parser::Error {
-                input: "1 > 2 == 3",
-                code: nom::error::ErrorKind::Satisfy,
-                message: Some(
-                    "Comparison and equality operators can't be mixed without parentheses"
-                        .to_string()
-                ),
-            }))
+            Ok((
```

Comment:
I'm a bit uncomfortable with that being an  `Ok`. It should be fine when integrated with other parsers that would bail on "==3" though

Reply:
Yes, chucking an eof() would make the parser fail.